src/magicbox/SuggestionsManager.ts

```diff
@@ -180,11 +180,13 @@ export class SuggestionsManager {
     $$(this.magicBoxContainer).setAttribute('aria-expanded', this.hasSuggestions.toString());
 
     if (!this.hasSuggestions) {
```

Comment:
Should there be a unit test checking that the attributes are present/removed depending on whether `hasSuggestions` is `true/false`?

Reply:
The accessibility tests cover that, if they would not be there it would show an error.